pkg/providers/baremetal/netq.go

```diff
@@ -0,0 +1,202 @@
+package baremetal
```

Comment:
Please add License header for the new files.
It could be 2 lines:
```
/*
 * Copyright 2025 NVIDIA CORPORATION
 * SPDX-License-Identifier: Apache-2.0
 */
```